From: Tim Bird <tim.bird@sony.com>
Subject: kernel: add SPDX-License-Identifier lines
Date: Mon, 19 Jan 2026 15:40:21 -0700

Add SPDX-License-Identifier lines to some old kernel files.

Link: https://lkml.kernel.org/r/20260119224021.793309-1-tim.bird@sony.com
Signed-off-by: Tim Bird <tim.bird@sony.com>
Acked-by: Karim Yaghmour <karim.yaghmour@opersys.com>
Cc: Jason Xing <kernelxing@tencent.com>
Cc: Peter Zijlstra <peterz@infradead.org>
Cc: Rusty "Buggy" Russell <rusty@rustcorp.com.au>
Cc: Thomas Gleixner <tglx@kernel.org>
Cc: Tim Bird <tim.bird@sony.com>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 kernel/cpu.c   |    3 +--
 kernel/relay.c |    3 +--
 2 files changed, 2 insertions(+), 4 deletions(-)

--- a/kernel/cpu.c~kernel-add-spdx-license-identifier-lines
+++ a/kernel/cpu.c
@@ -1,7 +1,6 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
 /* CPU control.
  * (C) 2001, 2002, 2003, 2004 Rusty Russell
- *
- * This code is licenced under the GPL.
  */
 #include <linux/sched/mm.h>
 #include <linux/proc_fs.h>
--- a/kernel/relay.c~kernel-add-spdx-license-identifier-lines
+++ a/kernel/relay.c
@@ -1,3 +1,4 @@
+// SPDX-License-Identifier: GPL-2.0
 /*
  * Public API and common code for kernel->userspace relay file support.
  *
@@ -9,8 +10,6 @@
  * Moved to kernel/relay.c by Paul Mundt, 2006.
  * November 2006 - CPU hotplug support by Mathieu Desnoyers
  * 	(mathieu.desnoyers@polymtl.ca)
- *
- * This file is released under the GPL.
  */
 #include <linux/errno.h>
 #include <linux/stddef.h>
_
